test/modules/encoders/test_cnn_encoder.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import torch
+from torch import nn, Tensor
+from test.test_utils import assert_expected
+from torchmultimodal.modules.encoders.cnn_encoder import CNNEncoder
```

Comment:
there should usually be some empty lines in between imports and class definition, did the pre-commit hooks run?